diff -rupN ui/default/footer.inc ui_no-newsfeed/default/footer.inc
--- ui/default/footer.inc	2018-05-15 23:40:46.000000000 +0200
+++ ui_no-newsfeed/default/footer.inc	2018-08-20 10:40:05.165726959 +0200
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 	
 	<div class="clear"></div>
diff -rupN ui/default/footer_minified.inc ui_no-newsfeed/default/footer_minified.inc
--- ui/default/footer_minified.inc	2018-05-15 23:40:46.000000000 +0200
+++ ui_no-newsfeed/default/footer_minified.inc	2018-08-20 10:40:15.829163886 +0200
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 </div> <!-- end #container -->
 	
diff -rupN ui/dressprow/footer.inc ui_no-newsfeed/dressprow/footer.inc
--- ui/dressprow/footer.inc	2018-05-15 23:40:55.000000000 +0200
+++ ui_no-newsfeed/dressprow/footer.inc	2018-08-20 10:40:24.645916301 +0200
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 	
 	<div class="clear"></div>
diff -rupN ui/dressprow/footer_minified.inc ui_no-newsfeed/dressprow/footer_minified.inc
--- ui/dressprow/footer_minified.inc	2018-05-15 23:40:55.000000000 +0200
+++ ui_no-newsfeed/dressprow/footer_minified.inc	2018-08-20 10:40:32.252657022 +0200
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 </div> <!-- end #container -->
 	
diff -rupN ui/phplist-ui-bootlist/footer_minified.inc ui_no-newsfeed/phplist-ui-bootlist/footer_minified.inc
--- ui/phplist-ui-bootlist/footer_minified.inc	2018-05-15 23:40:57.000000000 +0200
+++ ui_no-newsfeed/phplist-ui-bootlist/footer_minified.inc	2018-08-20 10:40:45.036114947 +0200
@@ -1,9 +1,4 @@
 <?php if (empty($_GET['page']) || $_GET['page'] == 'dashboard' || $_GET['page'] == 'home') { ?>
-	            <div class="clear"></div>
-	            <div id="newsfeed" class="menutableright block">
-					       <h3><span class="glyphicon glyphicon-bullhorn text-danger"></span> <?php print s('phpList community news'); ?></h3>
-								<?php  include 'communityfeed.php'; ?>
-		    </div>
 <?php } ?>
 	    </div> <!-- end #content -->
 	        <div id="footer">
