diff -rupN ui/default/footer.inc ui_no-newsfeed/default/footer.inc
--- ui/default/footer.inc	2019-02-11 14:51:42.000000000 +0100
+++ ui_no-newsfeed/default/footer.inc	2019-02-20 10:50:41.839802567 +0100
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 	
 	<div class="clear"></div>
diff -rupN ui/default/footer_minified.inc ui_no-newsfeed/default/footer_minified.inc
--- ui/default/footer_minified.inc	2019-02-11 14:51:42.000000000 +0100
+++ ui_no-newsfeed/default/footer_minified.inc	2019-02-20 10:50:41.839802567 +0100
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 </div> <!-- end #container -->
 	
diff -rupN ui/dressprow/footer.inc ui_no-newsfeed/dressprow/footer.inc
--- ui/dressprow/footer.inc	2019-02-11 14:51:55.000000000 +0100
+++ ui_no-newsfeed/dressprow/footer.inc	2019-02-20 10:50:41.839802567 +0100
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 	
 	<div class="clear"></div>
diff -rupN ui/dressprow/footer_minified.inc ui_no-newsfeed/dressprow/footer_minified.inc
--- ui/dressprow/footer_minified.inc	2019-02-11 14:51:55.000000000 +0100
+++ ui_no-newsfeed/dressprow/footer_minified.inc	2019-02-20 10:50:41.839802567 +0100
@@ -13,11 +13,6 @@
         <div id="language">
             <?php print $languageswitcher; ?>
         </div>
-	<div class="clear"></div>
-        <div id="newsfeed" class="menutableright block">
-            <h3><?php echo s('phpList community news'); ?></h3>
-            <?php include 'communityfeed.php'; ?>
-        </div>
 	</div><!-- end #right-bar -->
 </div> <!-- end #container -->
 	
diff -rupN ui/phplist-ui-bootlist/footer_minified.inc ui_no-newsfeed/phplist-ui-bootlist/footer_minified.inc
--- ui/phplist-ui-bootlist/footer_minified.inc	2019-02-11 14:52:02.000000000 +0100
+++ ui_no-newsfeed/phplist-ui-bootlist/footer_minified.inc	2019-02-20 10:50:41.843135922 +0100
@@ -1,10 +1,3 @@
-<?php if (empty($_GET['page']) || $_GET['page'] == 'dashboard' || $_GET['page'] == 'home') { ?>
-                <div class="clear"></div>
-                <div id="newsfeed" class="menutableright block">
-                           <h3><span class="glyphicon glyphicon-bullhorn text-danger"></span> <?php print s('phpList community news'); ?></h3>
-                                <?php  include 'communityfeed.php'; ?>
-            </div>
-<?php } ?>
         </div> <!-- end #Wrapp -->
             <div id="footer">
                 <div class="version pull-left">
